### ISSUE TITLE
Are Bluetooth connections encrypted?

### ISSUE THREAD

**Issue by floodoo** [PhilipsHue/flutter_reactive_ble#764]
Hello,

I’ve been developing an application using this package and have been considering the security of the Bluetooth connections. Specifically, I’m interested in understanding whether the connections established through your package are encrypted.

As per my understanding, Bluetooth does have built-in support for encryption, and this is generally handled automatically at the operating system level when two devices are paired. However, I couldn’t find specific information regarding this in the documentation for the Flutter Reactive BLE package.

Could you please confirm if the connections established using Flutter Reactive BLE leverage the encryption provided by the underlying operating system’s Bluetooth stack? If so, are there any specific requirements or steps we need to ensure or follow to enable this encryption?

Thank you in advance for your help!

**Comment by Taym95** [comment]
Yes Bluetooth does have built-in support for encryption, and this is generally handled automatically at the operating system level when two devices are paired, you confirm by checking OS pairing-pop that appears when you connect with a device using this library, we don't provide any extra encryption provided by the library.
